src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const thead = table.querySelector('thead');
+const tbody = table.querySelector('tbody');
+const tfoot = table.querySelector('tfoot');
+const thElements = document.querySelectorAll('table thead th');
+const positionTh = thElements[1];
+const tfootThElements = document.querySelectorAll('table tfoot th');
+const tfootTh = tfootThElements[1];
+
+const newTh = document.createElement('th');
+
+newTh.textContent = positionTh.textContent;
+thead.rows[0].insertBefore(newTh, thead.rows[0].lastElementChild);
+
+for (const row of tbody.rows) {
```

Comment:
Before iterating over 'tbody.rows', ensure that 'tbody' is not null and contains rows. This will prevent errors if the table body is empty or not present.